### DISCUSSION TITLE
"Current User" in table gets erased when running automation Update Row

### DISCUSSION THREAD

**Discussion by Bergem93** [Budibase/budibase#9766] [Help] [answered]
Hi,

EDIT: Solved the issue by deleting the column and reproduce it with a different name. Don't know the root cause.

I'm running an automation which updates rows which contain assets purchased more then 2,5 years ago. The update changes a value from "No" to "Yes". My problem is that it also clears the value for Current User, which is a relationship field. I have no idea why this is happening. I've also tried to run the Update Rows step without any changes in it, and it still clears the rows. 

The automation works as intended except for this issue. 

Any pointers?

Here is the automation:
![image](https://user-images.githubusercontent.com/5398191/220380001-88cb00b7-ade0-43f1-bcb3-41bf04fa5b6d.png)
![image](https://user-images.githubusercontent.com/5398191/220380060-cbd71d13-6411-4f84-8423-d7f5cf5d8da9.png)
![image](https://user-images.githubusercontent.com/5398191/220380113-3fadb98b-0e8c-4831-87e6-b87cc35b8316.png)

The table in question. As you can see the rows which were updated to "Less than 6 Months = Yes", Current User have been cleared, and therefore also Current User Email which is related. 
![image](https://user-images.githubusercontent.com/5398191/220380862-71fc6d51-8b77-4319-8b2c-668824918698.png)

**Comment by melohagan** [comment, marked as answer]
Hey @Bergem93 

The **Current User Email** - what type of field is that? Is it related to the **Current Owner** ? If so, I can see in your screenshot that you're not assigning a *Current Owner* value in the **Update Row** *Step 3*.

**Reply by Bergem93** [reply]
@melohagan 

Hi again! I managed to get it working by deleting the column and reproduce it with a different name. No clue what the issue was. 

Current User Email is a formula field which fetches email through Current User relationship field. `{{ Current Owner.0.email }}`
Current Owner is already populated before the automation is run, so the email gets fetched as intended. The issue was that the Current Owner value was cleared after running the automation, so data was essentially lost which is not a good thing.
